cmd/openshift-install/gather.go

```diff
@@ -197,3 +202,34 @@ func unSupportedPlatformGather(directory string) error {
 
 	return logGatherBootstrap(gatherBootstrapOpts.bootstrap, 22, gatherBootstrapOpts.masters, directory)
 }
+
+func logClusterOperatorConditions(ctx context.Context, config *rest.Config) error {
+	client, err := configclient.NewForConfig(config)
+	if err != nil {
+		return errors.Wrap(err, "creating a config client")
+	}
+
+	operators, err := client.ConfigV1().ClusterOperators().List(metav1.ListOptions{})
+	if err != nil {
+		return errors.Wrap(err, "listing ClusterOperator objects")
+	}
+
+	for _, operator := range operators.Items {
+		logger := logrus.WithField("operator", operator.ObjectMeta.Name)
+		for _, condition := range operator.Status.Conditions {
+			if (condition.Type == configv1.OperatorAvailable || condition.Type == configv1.OperatorUpgradeable) && condition.Status == configv1.ConditionTrue {
```

Comment:
The installer shouldn't care about Upgradeable.

Reply:
> The installer shouldn't care about Upgradeable.

Done with 7c7e4cf -> 618a84f.